packages/arb-token-bridge-ui/src/hooks/useBalance.ts

```diff
@@ -70,19 +71,28 @@ const useBalance = ({ provider, walletAddress }: UseBalanceProps) => {
         return {}
       }
 
-      const multiCaller = await MultiCaller.fromProvider(provider)
-      const addressesBalances = await multiCaller.getTokenData(_addresses, {
-        balanceOf: { account: walletAddressLowercased }
-      })
+      try {
```

Comment:
Wrapped multicall in a try-catch block and captured the exception on Sentry

Reply:
Implementing the fix or fallback will be part of the next PR once we get any data from logs.